cmd/k8s-operator/proxygroup.go

```diff
@@ -734,16 +734,19 @@ func (e *FindStaticEndpointErr) Error() string {
 	return e.msg
 }
 
-func (r *ProxyGroupReconciler) findStaticEndpoints(ctx context.Context, proxyGroup *tsapi.ProxyGroup, proxyClass *tsapi.ProxyClass, port uint16, logger *zap.SugaredLogger) ([]netip.AddrPort, error) {
-	currAddrs := []netip.AddrPort{}
-	for _, d := range proxyGroup.Status.Devices {
-		for _, a := range d.StaticEndpoints {
-			addr, err := netip.ParseAddrPort(a)
-			if err != nil {
-				logger.Debugf("failed to parse endpoint in status for ProxyGroup %q", proxyGroup.Name)
+func (r *ProxyGroupReconciler) findStaticEndpoints(ctx context.Context, existingCfgSecret *corev1.Secret, proxyClass *tsapi.ProxyClass, port uint16, logger *zap.SugaredLogger) ([]netip.AddrPort, error) {
+	var currAddrs []netip.AddrPort
+	if existingCfgSecret != nil {
+		oldConfB := existingCfgSecret.Data[tsoperator.TailscaledConfigFileName(106)]
+		if len(oldConfB) > 0 {
```

Comment:
is the embedded `if`'ing like this not the one? Happy to change it to something else if people want.